templates/service/systemd/mesos.service.erb

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Mesos <%= @name %>
+After=network.target
+Wants=network.target
+
+[Service]
+ExecStart=/usr/sbin/mesos-<%= @name %> <% if @name=='slave' -%>-containerizers=docker,mesos<% end -%>
```

Comment:
It should be a parameter (string) or maybe an array. 
Anyway you're trying to replace file that is already [part of a binary package](https://github.com/mesosphere/mesos-deb-packaging/blob/master/systemd/master.systemd) and bypassing the `mesos-init-wrapper` which would normally read all the properties from `/etc/mesos-master`. Any good reason why to do that?